app/controllers/cases_controller.rb

```diff
@@ -109,10 +109,19 @@ def filter
     redirect_to redirect_url
   end
 
+  def select_type
+    @case = Case.new
+  end
+
   def new
+    if FeatureSet.sars.enabled?
+      @type = params[:case][:type]
+    end
+
+
     authorize Case.new(category: Category.foi), :can_add_case?
 
-    @case = Case.new(type: nil)
+    @case = Case.new
```

Comment:
Is it worth adding a rescue if no case type is set?

Reply:
Sounds like it should raise an error to the user. As above, I think that we can handle that in CT-1391.